docs/fabrication/hand-tools/hammer.md

```diff
@@ -0,0 +1,13 @@
+# Hammers
+## Using A Hammer
```

Comment:
Uses for hammers.

This might need to go into nails vs screw? That can be another section though.

Also want to talk about using hammers to pull stuff out (like nails)

Reply:
Would be another section. Adding pulling stuff out.